glossary/en/terms/open-format/index.md

```diff
@@ -5,7 +5,7 @@ title: Open format
 ---
 
 <<<<<<< HEAD
-A [file format](/glossary/en/terms/file-format/) whose structure is set out in agreed [standards](/glossary/en/terms/standard), overseen and published by a non-commercial expert body. A file in an open format enjoys the guarantee that it can be correctly read by a range of different software programs or used to pass information between them. Compare [proprietary](/glossary/en/terms/proprietary/).
+A [file format](../file-format/) whose structure is set out in agreed [standards](../standard/), overseen and published by a non-commercial expert body. A file in an open format enjoys the guarantee that it can be correctly read by a range of different software programs or used to pass information between them. Compare [proprietary](../proprietary/).
 =======
```

Comment:
@jgkim - if I'm not mistaken, this is a conflict? Do you want me to fix it or you?